### DISCUSSION TITLE
<Link> using map Unhandled Runtime Error

### DISCUSSION THREAD

**Discussion by ullaskunder3** [vercel/next.js#44866] [Help] [answered]
I'm trying to build md blog preview when I next/link to linking using map it shows error
![image](https://user-images.githubusercontent.com/66258652/212389752-d6b98fff-bb1c-47a0-846d-e537bb00ff80.png)
if I use anchor tag it works file
![image](https://user-images.githubusercontent.com/66258652/212390191-0314d0cd-6de0-4b40-81ab-d165a79120d0.png)

need help what is it I'm doing wrong....

**Comment by icyJoseph** [comment]
What next.js version are you using? I can't really reproduce this with 13.1.1 

Could you try to remove node_modules, `.next`, and try again?

**Reply by ullaskunder3** [reply]
okkk,... I'll try that
next version is   "next": "^13.0.7",

**Comment by ullaskunder3** [comment, marked as answer]
Okk problem solved it was because of the version 13.0.7 upgrading it to "next": "^13.1.2", works fine thanks
